.changeset/smart-yaks-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Added a `z-index` to the NotificationToasts so they always float above other content.
```

Comment:
I would mention that this requires a bump of `design-tokens` (otherwise it's not obvious by looking just at the Circuit changelog)

Reply:
For example:

```suggestion
Added a `z-index` to the NotificationToasts so they always float above other content. Requires `@sumup/design-tokens@3.3.0`.
```